cmd/disk-cache-backend.go

```diff
@@ -214,11 +221,17 @@ func (c *diskCache) diskAvailable(size int64) bool {
 }
 
 // Purge cache entries that were not accessed.
-func (c *diskCache) purge() {
+func (c *diskCache) purge(ctx context.Context, doneCh <-chan struct{}) {
+	if c.diskUsageLow() {
+		return
+	}
+
+	// delete unaccessed objects older than expiry duration
+	expiry := UTCNow().Add(time.Duration(-1 * c.expiry))
```

Comment:
```suggestion
	expiry := UTCNow().Add(-c.expiry)
```

Probably leftovers from when it was untyped.

Reply:
fixed.